asd-aituber/apps/web/__tests__/https-setup-integration.test.ts

```
@@ -0,0 +1,516 @@
+      expect(result.steps.every(s => s.success)).toBe(true)
+    })
+
+    it('should handle various failure scenarios in setup flow', async () => {
+      // Test failure scenarios
+      const failureScenarios = [
+        {
+          name: 'mkcert not installed',
+          setup: () => {
+            testState.certificatesGenerated = false
+            mockExec.mockImplementation((cmd: string, cb: Function) => {
+              cb(new Error('mkcert: command not found'), '', '')
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)